.github/workflows/assets.yaml

```diff
@@ -93,12 +93,46 @@ jobs:
         build-args: |
           TAG=v${{ env.VERSION }}
 
+  gh-action:
```

Comment:
gh-action test needs to run after the release assets are created. Hence moving them.